x-pack/filebeat/module/azure/activitylogs/ingest/pipeline.yml

```diff
@@ -39,6 +39,7 @@ processors:
 - grok:
     field: azure.activitylogs.callerIpAddress
     patterns:
+      - "%{IPV6:source.ip}"
```

Comment:
I remember a previous PR where other people switched from the `grok` processor to the `covert` processor with a `type: ip`.

For example, https://github.com/elastic/integrations/pull/3411/files.

This looks like the recommended approach. Can you test if it also works for your use case?

Reply:
Seems like we might be able to just port those same changes from the elastic package to these pipelines. I think the behavior will slightly change since it can't be hostnames or ip:port or host:port pairs anymore, but as far as I'm aware this never happens in Azure logging, but it would be nice if I can find some documentation about that rather than assuming this is the case.

Reply:
Do you mean some documentation from the Azure side about the expected formats for these with IP addresses and ports?

Reply:
Hey @cFire, I am sorry. I see these Azure logs are using a variety of formats for the IP, so grok is better suited for this task. 

Let me double-check some more logs samples to see it this is the case.

Reply:
I am looking for some test documents because, according to [Grok processor reusable patterns](https://github.com/elastic/elasticsearch/tree/8.6/libs/grok/src/main/resources/patterns), the pattern named `IPORHOST ` [should already support IPv6](https://github.com/elastic/elasticsearch/blob/8.6/libs/grok/src/main/resources/patterns/legacy/grok-patterns#L30-L33).